flang/lib/Lower/OpenMP.cpp

```diff
@@ -178,14 +179,27 @@ genOMP(Fortran::lower::AbstractConverter &converter,
                          &clause.u)) {
         const Fortran::parser::OmpObjectList &ompObjectList = copyinClause->v;
         genObjectList(ompObjectList, converter, copyinClauseOperands);
-      }
+      } else if(const auto &allocateClause = 
+                  std::get_if<Fortran::parser::OmpAllocateClause>(&clause.u)){
+        if (const auto &allocatorValue = 
+              std::get<std::optional<Fortran::parser::OmpAllocateClause::Allocator>>(allocateClause->t)){
```

Comment:
Unsure of this formatting here. Did you clang-formatted it before submitting ?

Reply:
No, I missed that, I will do so when I update this review. Thanks for pointing it out.